apps/documentation/src/routes/documentation/configuration.mdx

```diff
@@ -55,3 +55,22 @@ Useful for extending vite's functionality
 
 - Type: `Array<Plugin | Array<Plugin> | Promise<Plugin | Array<Plugin>>`
 - [Vite documentation: plugins](https://vite.dev/guide/using-plugins)
+
+## server
+
+The following options are only affecting builds served from `tuono dev`.
```

Comment:
I think this should be more a quick `server` description rather than a specification. Also, the config updates also the production build.

What about:

```suggestion
The following options configure the server.
```